buildspec.yml

```diff
@@ -77,9 +77,11 @@ phases:
       - export TF_VAR_tfi_lx_userdata_log="${TFI_LX_USERDATA_LOG}"
       - export TF_VAR_tfi_s3_bucket="${TFI_S3_BUCKET}"
       - export TF_VAR_tfi_build_date=$(date +'%Y%m%d')
-      - export TF_VAR_tfi_build_id=$(date +'%H%M')"_"$(echo $CODEBUILD_BUILD_ID | cut -d'-' -f 5)
-      - #export TF_VAR_tfi_build_id=$(echo $CODEBUILD_BUILD_ID | cut -d'-' -f 5)
-      - export TF_VAR_tfi_instance_name_tag="terrafirm-"$(echo $CODEBUILD_BUILD_ID | cut -d'-' -f 5)
+      - # real cb env looks like "<build project>:fb1b0f16-3863-4cdd-9251-4b3efe3d6796" - here using last chunk (eg, 4b3efe3d6796) for shortness
```

Comment:
`fb1b0f16-3863-4cdd-9251-4b3efe3d6796` is a uuid, let's shorten the helper to `"<build project>:<build uuid>"`